### PR TITLE
*: add descriptions for some system tables

### PR DESCRIPTION
**Pull request by wjhuang2016**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: mysql-schema.md [hunk 1]
```diff
@@ -71,14 +71,28 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
...
-* `tidb_mdl_view`：元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
-* `tidb_mdl_info`：TiDB 内部用于同步各节点的元数据锁的相关信息
+* `tidb_mdl_view` 元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
+* `tidb_mdl_info` TiDB 内部用于同步各节点的元数据锁的相关信息
+
+## DDL 相关系统表
+
+* `tidb_ddl_history` 记录了 DDL 语句的历史记录。
+* `tidb_ddl_jobs` TiDB 内部存放的正在执行 DDL 的元数据，用于执行 DDL。
+* `tidb_ddl_reorg` TiDB 内部存放的正在执行的物理 DDL（例如加索引）的元数据，用于执行物理 DDL。
```
Comment (hfxsd) [inline mysql-schema.md:87]:
```suggestion
* `tidb_ddl_history` 记录了 DDL 语句的历史记录
* `tidb_ddl_jobs` TiDB 内部存放的正在执行 DDL 的元数据，用于执行 DDL
* `tidb_ddl_reorg` TiDB 内部存放的正在执行的物理 DDL（例如加索引）的元数据，用于执行物理 DDL
```

File: mysql-schema.md [hunk 2]
```diff
@@ -71,14 +71,28 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
...
+* `tidb_mdl_view` 元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
+* `tidb_mdl_info` TiDB 内部用于同步各节点的元数据锁的相关信息
+
+## DDL 相关系统表
+
+* `tidb_ddl_history` 记录了 DDL 语句的历史记录
+* `tidb_ddl_jobs` TiDB 内部存放的正在执行 DDL 的元数据，用于执行 DDL
+* `tidb_ddl_reorg` TiDB 内部存放的正在执行的物理 DDL（例如加索引）的元数据，用于执行物理 DDL
+
+## 分布式框架相关系统表
```
Comment (hfxsd) [inline mysql-schema.md:89]:
```suggestion
## 分布式执行框架相关系统表
```

File: mysql-schema.md [hunk 3]
```diff
@@ -71,14 +71,28 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
...
+
+* `dist_framework_meta` 存放后台任务调度的元信息
+* `tidb_global_task` 存放当前正在执行的分布式任务元信息
+* `tidb_global_task_history` 存放历史的分布式任务元信息
+* `tidb_background_subtask` 存放当前正在执行的分布式任务的子任务元信息
+* `tidb_background_subtask_history` 存放历史的分布式任务的子任务元信息
```
Comment (hfxsd) [inline mysql-schema.md:95]:
```suggestion
* `dist_framework_meta` 存放分布式执行框架任务调度的元信息
* `tidb_global_task` 存放当前正在执行的分布式执行框架任务元信息
* `tidb_global_task_history` 存放历史的分布式执行框架任务元信息
* `tidb_background_subtask` 存放当前正在执行的分布式执行框架任务的子任务元信息
* `tidb_background_subtask_history` 存放历史的分布式执行框架任务的子任务元信息
```

**Review by hfxsd — APPROVED** [review]

**Review by okJiang — APPROVED** [review]

File: mysql-schema.md [hunk 4]
```diff
@@ -71,14 +71,28 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
...
+## DDL 相关系统表
+
+* `tidb_ddl_history` 记录了 DDL 语句的历史记录
+* `tidb_ddl_jobs` TiDB 内部存放的正在执行 DDL 的元数据，用于执行 DDL
+* `tidb_ddl_reorg` TiDB 内部存放的正在执行的物理 DDL（例如加索引）的元数据，用于执行物理 DDL
+
+## 分布式执行框架相关系统表
+
+* `dist_framework_meta` 存放分布式执行框架任务调度的元信息
+* `tidb_global_task` 存放当前正在执行的分布式执行框架任务元信息
```
Comment (Benjamin2037) [inline mysql-schema.md:92]:
```suggestion
* `tidb_global_task` 存放当前分布式框架正在执行的任务元信息
```

File: mysql-schema.md [hunk 5]
```diff
@@ -71,14 +71,28 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
...
+* `tidb_global_task_history` 存放历史的分布式执行框架任务元信息
```
Comment (Benjamin2037) [inline mysql-schema.md:93]:
```suggestion
* `tidb_global_task_history` 存放分布式执行框架完成（成功或者失败）的任务元信息
```

File: mysql-schema.md [hunk 6]
```diff
@@ -71,14 +71,28 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
...
 
-* `tidb_mdl_view`：元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
-* `tidb_mdl_info`：TiDB 内部用于同步各节点的元数据锁的相关信息
+* `tidb_mdl_view` 元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
+* `tidb_mdl_info` TiDB 内部用于同步各节点的元数据锁的相关信息
+
+## DDL 相关系统表
+
+* `tidb_ddl_history` 记录了 DDL 语句的历史记录
+* `tidb_ddl_jobs` TiDB 内部存放的正在执行 DDL 的元数据，用于执行 DDL
```
Comment (Oreoxmt) [inline mysql-schema.md:86]:
```suggestion
* `tidb_ddl_jobs` TiDB 内部存放的正在执行的 DDL 的元数据，用于执行 DDL
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Pull request merged**